notes/stories/film/film-scifi-2000s.st.txt

```diff
@@ -497,6 +497,51 @@ cryptid [A scientist explained that the strange insects were actually prehistori
 speculative vehicle [The Fighter Griffon flying vehicle could fly like a jet but also hover like a helicopter.]
 
 
+movie: Red Planet (2000)
+========================
+
+:: Title
+Red Planet
+
+:: Description
+Red Planet is a 2000 science fiction action film directed by Antony Hoffman,
+starring Val Kilmer, Carrie-Anne Moss, and Tom Sizemore.
+
+Synopsis: A pioneering team of astronauts are sent to Mars as a prelude to
+colonization, as the Earth is dying from an ecological crisis.
+
+:: Date
+2000-11-10
+
+:: References
+https://en.wikipedia.org/wiki/Red_Planet_(film)
+
+:: Ratings
+2 <paul>
+
+:: Major Themes
+colonization of Mars [The central premise of the story is that the Earth was dying and the founding of a colony of Mars was the only hope for humanity.]
+speculative terraforming [A key element of the plot is that humanity had seeded Mars with atmosphere-producing algae as the first stage of terraforming the planet.]
+ecological destruction [The story is set a near future where Earth is in a state of ecological crisis and the only hope for humanity lies in the founding of a colony on Mars.]
+human vs. nature [Much of the film concerned the provision-less protagonists as they walked across the surface of Mars to reach an old Russian probe's sample-return module which they would use to launch themselves into orbit and to safety. They overcame a violent ice storm among other things along the way.]
+human vs. technology [The protagonists were stalked by the advanced, but non-sentient military robot AMEE as they trekked across the surface of Mars.]
+
+:: Minor Themes
+overpopulation [It was stated in the opening narration that the Earth had become overpopulated by the year 2000.]
+pollution [It was stated in the opening narration that humanity had "polluted and poisoned" the Earth beyond the point where it could be cleaned up.]
+space station [The MARS-1 crew departed on their mission to the Red Planet from a large space station situated in high Earth orbit.]
+interplanetary space travel [The first part of the film concerned followed Commander Kate Bowman and her crew as they went on the first manned mission to Mars.]
+speculative spaceship [The protagonists flew to Mars in an experimental, fusion powered spaceship.]
+speculative robot [The anthropomorphic, autonomous military robot AMEE went haywire and started stalking the protagonists.]
```

Comment:
https://www.themeontology.org/theme.php?name=autonomous%20weapons%20system ?

Reply:
I added it. But I suppose we should consider at some point whether this should apply for individual robots or just for larger systems, like that one from TNG The Arsenal of Freedom.